packages/backend/tsoa.json

```diff
@@ -12,6 +14,10 @@
         },
         "license": "MIT",
         "tags": [
+            {
```

Comment:
I know is not as clean but we should have this in the `info.description` instead of `tags`.

We can use markdown to create sections 
```
"description": "Open API documentation for all public Lightdash API endpoints. \n # Authentication\nBefore you get started, you might need to create a Personal Access Token to authenticate via the API. You can create a token by following this guide: https://docs.lightdash.com/references/personal_tokens",
        
```
<img width="1717" alt="Screenshot 2024-01-15 at 17 24 31" src="https://github.com/lightdash/lightdash/assets/9117144/16b1bfdd-4b9e-4f66-b6d4-caa34a1a7102">

Reply:
We could also convert `tsoa.json` to `tsoa.yml` and update the command to `tsoa spec-and-routes --configuration tsoa.yml` 
This way we can have the description over multiple lines.

Reply:
thanks for both suggestions, will apply them soon!